packages/icon/src/glyphCommon.ts

```diff
@@ -14,15 +14,44 @@ export const sizeMap: Record<Size, number> = {
   xlarge: 24,
 } as const;
 
-export function getGlyphTitle(name: string, title?: string | boolean | null) {
-  if (title === false) {
-    // If title is null, we unset the title entirely, otherwise we generate one.
-    return null;
-  }
+interface AccessibleFunctionParams {
+  title?: string | null;
+  ariaLabelledby?: string;
+  ariaLabel?: string;
+}
```

Comment:
delete

Reply:
can't delete bc we use camel case spelling for the inputs and then kebab case for the outputs, but renamed